setup.py

```diff
@@ -459,20 +463,43 @@ def build_tf_extension(build_ext, options):
     return [flag for flag in tf_compile_flags if '_GLIBCXX_USE_CXX11_ABI' in flag]
 
 
+def parse_version(version_str):
+    m = re.match('^(\d+)(?:\.(\d+))?(?:\.(\d+))?(?:\.(\d+))?', version_str)
+    if m is None:
+        return None
+
+    # turn version string to long integer
+    version = int(m.group(1)) * 10 ** 9
```

Comment:
I believe this is just going to give you a 32-bit integer.  You might want to do this instead:

`version = long(m.group(1)) * 10L ** 9`

Reply:
Doesn't look like Python cares about it:
```
>>> int(1000)*10**9
1000000000000
```

Reply:
I see, apparently they unified the `int` and `long` types into some time back.